### ISSUE TITLE
sorting of topics in topicprio

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#1629]
sorting in topicprio is automatically. If using numbers, 10 is listed between 0 and 1 but should of course come after 9. Following project was struggling with it: https://mein.berlin.de/projects/diskutieren-sie-das-impulspapier-fur-eine-charta-f/
Daniel sent me a link with possible solutions: http://ieao.de/plansprache/abc-wertrichtig.htm

I think it would also make sense to allow initiators their own sorting, but I guess this would be a story? As it might take longer to implement, maybe we still do the bugfix first?

**Comment by fuzzylogic2000** [comment]
This is not really a bug and not really a bugfix, I would say. The thing Daniel showed you doesn't seem any easier as just letting the initiators do their own sorting. Until then it might be a solution to always write 01, and 02?! Otherwise, if you alphabetically sort sth, it works like that and I can't see an easy way to change that.
I will remove the label bug and add the label US. @MagdaN Do you agree or know an easy fix?

**Comment by fuzzylogic2000** [comment]
or like that: https://stackoverflow.com/questions/2969348/django-ordering-numerical-value-with-order-by ?

**Comment by fuzzylogic2000** [comment]
Ok, I found a solution, which will order like that:
![screenshot from 2018-10-25 17-08-55](https://user-images.githubusercontent.com/8178179/47510704-f56c7100-d878-11e8-8b89-b7dd6487bd3a.png)

Before I invest more time to find all the places, where this should be replaced, could you say if you're fine with that ordering, @CarolingerSeilchenspringer 
(I noticed the case-sensitive-ordering of the z will also fix that!)

**Comment by fuzzylogic2000** [comment]
Oh no, I didn't find a solution. The problem was the 2! It still comes after the 10. So this didn't help. Sorry!

**Comment by fuzzylogic2000** [comment]
@CarolingerSeilchenspringer I now already spent an hour on it without finding a solution. As you called it bug, it sounded like it was kind of important to you. But I don't think, we can find a (quick) solution and also am not sure, that we can define an ordering that everyone will be fine with. (In alphabetical ordering, do the numbers come before or after the characters?, ...)
How important is it, that the alphabetical ordering will order the numbers by numbers and the words really alphabetically? Should I spent more time on it or wouldn't a US for the settable ordering the better solution anyway?
